metadata_backend/api/handlers.py

```diff
@@ -234,15 +249,15 @@ async def _extract_xml_upload(req: Request, extract_one: bool = False
         if extract_one and files:
             reason = "Only one file can be sent to this endpoint at a time."
             raise web.HTTPBadRequest(reason=reason)
-        xml_type = part.name.lower()
-        if xml_type not in object_types:
-            raise web.HTTPBadRequest(reason="Not ok type")
+        type = part.name.lower()
```

Comment:
i think `type` is a reserved name in python so was maybe:
```suggestion
        schema_type = part.name.lower()
```